sw/device/silicon_creator/lib/drivers/spi_device.h

```diff
@@ -359,6 +383,23 @@ void spi_device_flash_status_clear(void);
 OT_WARN_UNUSED_RESULT
 uint32_t spi_device_flash_status_get(void);
 
+/**
+ * Enables the SPI mailbox at the given SPI address.
+ *
+ * @param address The SPI address of the mailbox.
+ */
+void spi_device_enable_mailbox(uint32_t address);
+
+/**
+ * Copies data to the SPI egress buffer.
+ *
+ * @param egress_offset Destination offset in the egress buffer.
+ * @param data Pointer to data to copy into the egress buffer.
```

Comment:
Does the data need to be aligned to uint32_t boundaries?

Reply:
Yes; fixed the doc comment.